CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ In versions 1.3.1+, installing a new version of Chronograf automatically clears
 
 ### UI Improvements
   1. [#1451](https://github.com/influxdata/chronograf/pull/1451): Refactor scrollbars to support non-webkit browsers
+  1. [#1453](https://github.com/influxdata/chronograf/pull/1453): Increase the query builder's default height in cell editor mode and in the data explorer
```

Comment:
this change actually came in from a rebase, but yeah I agree it is no simple UI improvement